src/llama_stack_provider_kft/kft_adapter.py

```diff
@@ -324,6 +335,61 @@ async def get_training_job_artifacts(
             )
         return None
 
+    def create_init_containers(
+        self, resources_per_worker, env_vars
+    ) -> list[models.V1Container]:
+        dp_master_args = [
+            f"""
+            echo "Processing data"
+            echo "Using "{self.config.model_path}" model for training"
+            echo "Using "{self.config.data_path}"  data for training"
+
+            pip install instructlab-training==0.7.0
```

Comment:
Why do we install and pin to this version? Isn't the package part of the image already?

Reply:
@leseb see https://github.com/opendatahub-io/llama-stack-provider-kft/pull/18#discussion_r2024784710.